src/module/actor/character/sheet.ts

```diff
@@ -238,6 +238,7 @@ export class CharacterSheetPF2e extends CreatureSheetPF2e<CharacterPF2e> {
                 itemData.img ||= CONST.DEFAULT_TOKEN;
                 itemData.assetValue = item.assetValue;
                 itemData.isInvestable = isEquipped && isIdentified && isInvested !== null;
+                itemData.isDamaged = itemData.data.hp.value > 0 && itemData.data.hp.value < itemData.data.hp.max;
```

Comment:
Can this be a getter on `PhysicalItemPF2e`? These are going away (inventory items are now using Item class instances).

Reply:
Good idea!